src/main/resources/spring/camel-context.xml

```diff
@@ -51,6 +65,53 @@
             </get>
         </rest>
 
+        <rest path="/administration" produces="application/octet-stream">
+            <get uri="/report/csv">
+                <route id="administration-report-csv">
+                    <setHeader headerName="CSVCorrelationID">
+                        <method ref="utils" method="generateUUID"/>
+                    </setHeader>
+                    <setHeader headerName="unixEpochDate">
+                        <method ref="dateUtils" method="getUnixEpochDate"/>
+                    </setHeader>
+                    <setHeader headerName="nowDate">
+                        <simple>${date:now}</simple>
+                    </setHeader>
+                    <setHeader headerName="fromDate">
+                        <method ref="dateUtils" method="getLastMondayMinus7days"/>
+                    </setHeader>
+                    <setHeader headerName="toDate">
+                        <method ref="dateUtils" method="getLastMonday"/>
+                    </setHeader>
+
+                    <log message="Creating csv first file from date ${date:header.fromDate} to ${date:header.toDate}" />
+                    <bean ref="analysisService" method="getAdministrationMetrics(${header.fromDate}, ${header.toDate})"/>
+                    <bean ref="administrationMetricsMapper" method="toAdministrationMetricsModels"/>
+                    <to uri="direct:administrationMetricsModelToCsv"/>
+                    <setHeader headerName="CamelFileName">
+                        <simple>${date:header.fromDate:yyyyMMddHHmmssz}-to-${date:header.toDate:yyyyMMddHHmmssz}.csv</simple>
+                    </setHeader>
+                    <to uri="direct:administration-report-csv-aggregator" />
+
+                    <log message="Creating csv second file from date ${date:header.fromDate} to ${date:header.toDate}" />
+                    <bean ref="analysisService" method="getAdministrationMetrics(${header.unixEpochDate}, ${header.nowDate})"/>
+                    <bean ref="administrationMetricsMapper" method="toAdministrationMetricsModels"/>
+                    <to uri="direct:administrationMetricsModelToCsv"/>
+                    <setHeader headerName="CamelFileName">
```

Comment:
Use `Exchange.FILE_NAME` instead of its value `CamelFileName`

Reply:
Done